src/VbsEnclaveSDK/src/veil_host_lib/logger.vtl0.h

```diff
@@ -49,9 +49,9 @@ namespace veil::vtl0
                 return modified;
             }
 
-            void SetLogFilePath()
-            {
-                logFilePath = L"c:\\VeilLogs\\" + ReplaceForbiddenFilenameChars(CreateTimestamp()) + L".txt";
+            void SetLogFilePath()  
+            {  
+               logFilePath = std::filesystem::current_path().wstring() + L"\\" + ReplaceForbiddenFilenameChars(CreateTimestamp()) + L".txt";  
```

Comment:
>std::filesystem::current_path() [](http://example.com/codeflow?start=29&length=31)

so, the problem here is that this could now throw an exception depending on where the program is run from, whereas before it deterministically required admin to write to c:\veillog

(what if the user launches from a tile, what is the CWD? c:\program files? not even admin can write there)

if we're going to fix this anyway, then why not let the ctor take an argument that specifies the path?